modules/support-on-various-platforms.adoc

```diff
@@ -0,0 +1,130 @@
+
+| https://access.redhat.com/support/policy/updates/errata[Red Hat Enterprise Linux]
+| 7.3  till 7.9, 8.0 and newer
+
+| https://www.centos.org/centos-linux/[CentOS]
+| 7, 8
+
+| https://getfedora.org/en/coreos?stream=stable[Fedora CoreOS]
+| Stable stream 32.20200824.3.0 and newer
+
+| https://www.flatcar-linux.org/releases/[Flatcar Container Linux]
+| 2023.4.0 and newer
+
+| https://cloud.google.com/container-optimized-os/docs/release-notes#current_active_releases[Google COS]
+| 77 and newer
+
+| https://aws.amazon.com/amazon-linux-ami[Amazon Linux]
+| 2
+
+| https://github.com/gardenlinux/gardenlinux[Garden Linux]
+| 27.0 and newer
+|===
+
+[id="container-runtimes_{context}"]
+== Container runtimes
+
+|===
+| Container runtimes | Version
+
+| https://docs.docker.com/engine/release-notes/[Docker]
+| 17.03 and newer
+
+| https://github.com/cri-o/cri-o/releases[CRI-O] and https://github.com/opencontainers/runc/releases[runC]
+| -
+|===
+
+[NOTE]
+====
+CRI-O and runC support is available starting from {product-title} version 2.5.31.0.
+====
+
+[id="container-orchestrators-and-platforms_{context}"]
+== Container orchestrators and platforms
+
+|===
+| Container orchestrators and platforms | Version
+
+| https://kubernetes.io/docs/setup/release/notes/[Kubernetes]^1^
+| 1.15 and newer
+
+| https://access.redhat.com/support/policy/updates/openshift[{ocp}]
+| 3.10, 3.11, 4.1 and newer
+
+| https://docs.d2iq.com/mesosphere/dcos/services/kubernetes/[DC/OS Kubernetes]^2^
+| 2.0.0 and newer
+|===
+
+[NOTE]
+====
+
+. StackRox support new versions of Kubernetes within 3 months of their open-source general-availability release.
+You may need to upgrade the {product-title} to get support for the latest Kubernetes versions.
+See the <<kubernetes-support,Kubernetes support>> section for more details.
+. DC/OS support is available starting from {product-title} version 2.5.31.0.
+====
+
+[id="kubernetes-support_{context}"]
+=== Kubernetes support
+
+The following table lists the minimum version of {product-title} StackRox support based on the Kubernetes version.
+
+|===
+| Kubernetes version | StackRox minimum support version
+
+| 1.15, 1.16, 1.17
+| 3.0.42.0
+
+| 1.18
+| 3.0.47.1
+
+| 1.19
+| 3.0.52.0
+|===
+
+[id="managed-kubernetes-services_{context}"]
+== Managed Kubernetes services
+
+StackRox support recent Kubernetes and {ocp} versions, and test on managed Kubernetes service from all major cloud providers, including:
+
+* https://aws.amazon.com/eks/[Amazon Elastic Kubernetes Service (EKS)]
+* https://azure.microsoft.com/en-us/services/kubernetes-service/[Azure Kubernetes Service (AKS)]
+* https://cloud.google.com/kubernetes-engine/[Google Kubernetes Engine (GKE)]
+
+[NOTE]
+====
+
+Collector will not install on GKE clusters if you enable https://cloud.google.com/kubernetes-engine/docs/how-to/shielded-gke-nodes#secure_boot[secure boot] because third-party unsigned (not signed by Google's CA) kernel modules cannot be loaded when secure boot is enabled.
```

Comment:
I would avoid the parentheses and say something like, "...third-party unsigned kernel module, which are not signed by Google's CA, cannot be loaded..."